### DISCUSSION TITLE
NTSC 60 fps, VDC output?

### DISCUSSION THREAD

**Discussion by Barkfin** [frntc/RAD-Doom#1] [Ideas]
Hey there's a new video from "8 bit theory" regarding near-EGA graphics on the C128 using the VDC.
https://www.youtube.com/watch?v=ES4eBcKxmE4

He shows a method (in BASIC) for down-sampling an EGA graphic for output on C128's VDC, looks quite a bit like the VIC-II rendering being done here.
What would it take to make a 60 fps version? I just got my RAD kit from Restore-Store (28.35 € from Germany). I still have to solder the pins on and test if it will accept my R-Pi 4B so I'm a little while out from firing it up yet.

**Comment by frntc** [comment]
Hey,

the image conversion (full RGB to VDC-like) is less of a problem as it would run on the RPi. But sending data to the VDC is horribly slow since it's accessible via two registers only (and writing to those even requires a bit of a delay). In contrast, the VIC2 accesses the RAM to which the RAD can write at about 1mb/sec... (the Sidekick64 menu outputs to VIC2 and VDC in parallel if you want to have a look at the code).

Regarding your RAD: you will need a RPi 3A+/3B+ or Zero 2. Other RPis won't work!

**Reply by V3771n9** [reply]
Will it be compatible with c128 ntsc using vic2?

**Reply by frntc** [reply]
RAD-Doom is only for PAL-machines thus far...
